src/wallet/components/WalletDropdownFundLink.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { expect, it, vi } from 'vitest';
+import { version } from '../../version';
+import type { WindowSizes } from '../types';
+import { WalletDropdownFundLink } from './WalletDropdownFundLink';
+
+const FUNDING_URL = `http://keys.coinbase.com/fund?dappName=&dappUrl=http://localhost:3000/&version=${version}&source=onchainkit`;
```

Comment:
Lets put this const in /src/wallet/constants.ts

https://github.com/coinbase/onchainkit/blob/main/src/wallet/constants.ts